src/panels/developer-tools/service/developer-tools-service.ts

```diff
@@ -250,17 +251,10 @@ class HaPanelDevService extends LitElement {
   );
 
   private _callService() {
-    const domain = computeDomain(this._serviceData!.service);
-    const service = computeObjectId(this._serviceData!.service);
-    if (!domain || !service) {
+    if (!this._serviceData!.service) {
       return;
     }
-    this.hass.callService(
-      domain,
-      service,
-      this._serviceData!.data,
-      this._serviceData!.target
-    );
+    callServiceAction(this.hass, this._serviceData!);
```

Comment:
```suggestion
    callServiceAction(this.hass, this._serviceData);
```

Reply:
This change made the linter fail though. Should `callServiceAction` be updated to handle `undefined`?